client/login/magic-login/style.scss

```diff
@@ -288,6 +288,18 @@
 	}
 }
 
+.studio-magic-login__tos {
+	font-size: $font-body-extra-small;
+	margin: 20px 0 20px;
+	text-align: left;
+
+	a {
+		@include breakpoint-deprecated( ">480px" ) {
```

Comment:
Do we know why this mixin is deprecated? Is there an alternative we should use instead?

Reply:
Thank you for pointing this out, David! I found in https://github.com/Automattic/wp-calypso/pull/41985 that this was replaced with `@include break-mobile`, which I see is used throughout this CSS file too. Updated in https://github.com/Automattic/wp-calypso/pull/90204/commits/483625912c31dd4b537bb1f9b5a2f8ad868d52d1.